src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
 	}
+
 	var args = text.split(" ");
-	var cmd = args.shift().replace("/", "").toLowerCase();
+	var cmd = args.shift().toLowerCase();
+	var result;
+
 	_.each(inputs, function(plugin) {
 		try {
 			var path = "./plugins/inputs/" + plugin;
 			var fn = require(path);
-			fn.apply(client, [
+			result = fn.apply(client, [
 				target.network,
 				target.chan,
 				cmd,
 				args
 			]);
+			// Commands return true on successful execution
+			// lodash expects `false` to stop the loop
+			return !result;
 		} catch (e) {
 			console.log(path + ": " + e);
 		}
 	});
+
+	if (result !== true) {
+		target.network.irc.write(text);
```

Comment:
Yes! This is the magic sauce that will let things like `/znc` work! :raised_hands: